app/main.py

```diff
@@ -1,8 +1,51 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int, clean_mark: int, brand: str):
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: int, clean_power: int, average_rating: float, count_of_ratings: int):
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = round(average_rating, 1)
+        self.count_of_ratings = count_of_ratings
+
+    def calculate_washing_price(self, car: Car) -> float:
+        if car.clean_mark < self.clean_power:
+            cleanliness_difference = self.clean_power - car.clean_mark
+            base_price_multiplier = car.comfort_class * self.average_rating
+            single_car_cost = base_price_multiplier * cleanliness_difference / self.distance_from_city_center
+            return round(single_car_cost, 1)
+        return 0.0
```

Comment:
According to the checklist, you can avoid else when you have a return statement. You can simplify your method by returning the result of the calculation immediately if the condition is met, and returning 0.0 as a default at the end of the method.